.vscode/bevy.code-snippets

```diff
@@ -0,0 +1,75 @@
+{
+    "Bevy: New module-level function Plugin": {
+        "scope": "rust",
+        "prefix": "plugin_fn",
+        "body": [
+            "use bevy::prelude::*;",
+            "",
+            "pub(super) fn plugin(app: &mut App) {",
+            "\t$0",
```

Comment:
what does `$0` do for a prefix?

Reply:
That's where the cursor ends up after you Tab through all the placeholder locations (`$1, $2, ...`).

Reply:
TIL